### DISCUSSION TITLE
How does Owncast work?

### DISCUSSION THREAD

**Discussion by brown7477** [owncast/owncast#442] [General Owncast discussion]
I am curious how Owncast actually works. 
Is there an admin interface? 
Does it support multiple streams at 1 time? 
Can it be integrated into a social network to allow users to live stream?
I tried to watch a few livestreams that were posted but it said they were ended. But they were still there.
I really liked what I seen though so far.

P.S. I am not a coder like y'all, so please talk in Laymans english. LOL

**Comment by gabek** [comment]
> I tried to watch a few livestreams that were posted but it said they were ended. But they were still there.

For an example stream you can check out our demo that's running: https://watch.owncast.online/

> P.S. I am not a coder like y'all, so please talk in Laymans english. LOL

While you don't need to be a developer to use Owncast, it is a self-hosted piece of software that you're responsible for running yourself.  If you don't feel comfortable running your own server on the internet then Owncast might not be a good fit for you at the moment.

> Is there an admin interface?

There is not currently an interface where you can configure or modify your Owncast instance.  We'll shortly be releasing a version that will have an interface where you can view the status of your instance, but it's purely for information and monitoring purposes.  You still need to configure the service manually through a config file.  But having a way to manage your server through the admin will be coming in the future.

> Does it support multiple streams at 1 time?

It supports different quality streams of the same source content, but multiple people can't stream on a single Owncast server, for example.  It's not for building your own Twitch where you have users and accounts, but more for a person to run a live stream server that they control for themselves.

> Can it be integrated into a social network to allow users to live stream?

Social networks lock themselves down and you can't just insert your stream into their content unfortunately.  They want you to use their own live stream features.  Owncast is meant to live outside of all of that.

Hopefully this answers your questions!

**Reply by brown7477** [reply]
So this is an ideal fit for embedding into a website for a TV channel or a podcast.
I actually like this. It has some awesome potential and a lot of uses and I may have a project in mind for this.

**Reply by gabek** [reply]
Those sound like great uses!  Let us know if you have any other questions.
